.github/workflows/ci.yml

```diff
@@ -77,3 +77,58 @@ jobs:
           TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
           TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
         run: tox
+
+  publish-docker:
+    needs: [tests]
```

Comment:
Should it just run on `push`? Otherwise whenever someone makes a PR that passes tests it'll publish.

You probably want a global 
```yaml
    if: github.event_name == 'push'
```
here